pykafka/connection.py

```diff
@@ -63,7 +76,8 @@ def connect(self, timeout):
         """Connect to the broker."""
         self._socket = socket.create_connection(
             (self.host, self.port),
-            timeout=timeout / 1000
+            timeout / 1000,
+            self.source_host, self.source_port
```

Comment:
[They don't appear to be kwargs](https://docs.python.org/2/library/socket.html#socket.create_connection)